intelmq/bots/experts/http_status/expert.py

```diff
@@ -0,0 +1,59 @@
+"""
+HTTP Status Expert Bot
+
+SPDX-FileCopyrightText: 2021 Birger Schacht <schacht@cert.at>
+SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+from typing import List
+
+from intelmq.lib.bot import Bot
+from intelmq.lib.utils import create_request_session
+
+
+class HttpStatusExpertBot(Bot):
+    """
+    Fetch the HTTP Status for a given URI
+
+    Parameters
+    ----------
+    field: str
+        The name of the field containing the URL to be checked (required).
+    success_status_codes: List
+        A list of success status codes. If this parameter is omitted or the list is empty,
+        successful status codes are the ones between 200 and 400.
+    """
+    field: str = None  # The field containing the URI
+    success_status_codes: List[int] = []  # A list of status codes for success
+
+    def init(self):
+        if self.field is None:
+            raise ValueError('No fieldname provided.')
+
+    def process(self):
+        event = self.receive_message()
+
+        if self.field in event:
+            self.set_request_parameters()
+            session = create_request_session(self)
+
+            event.add('status', 'offline')
+
+            try:
+                response = session.get(event[self.field])
+                # If success_status_codes are configured, we use those
+                # to check the success of the request, otherwise we
+                # rely on the boolean value of the response
+                if (self.success_status_codes and response.status_code in self.success_status_codes) or (response):
+                    event.add('status', "online", overwrite=True)
```

Comment:
```suggestion
                    event.add('status', "online", overwrite=self.overwrite)
```
And add an overwrite parameter

Reply:
done.